docs/sources/enterprise/settings-updates.md

```diff
@@ -0,0 +1,79 @@
++++
+title = "Settings updates at runtime"
+description = "Settings updates at runtime"
+keywords = ["grafana", "runtime", "settings"]
+weight = 500
++++
+
+# Settings updates at runtime
+
+> **Note:** Available in Grafana Enterprise v8.0+.
+
+Settings updates at runtime allows you to update Grafana settings with no need to restart the instance.
+
+Those updates that happen at runtime are stored in the database and override settings from the other sources
+(arguments, environment variables, settings file, etc). Therefore, every time a specific setting key is removed at runtime,
```

Comment:
Would be nice to add links to these config methods, in case people don't know what they are.

Reply:
Yeah, great idea! Do you think a link to the general configuration page would be enough?
https://grafana.com/docs/grafana/latest/administration/configuration/

Or would be better to add a specific link to every alternative?
- https://grafana.com/docs/grafana/latest/administration/configuration/#config-file-locations
- https://grafana.com/docs/grafana/latest/administration/configuration/#configure-with-environment-variables
- https://grafana.com/docs/grafana/latest/administration/configuration/#variable-expansion
- https://grafana.com/docs/grafana/latest/administration/cli/#override-a-configuration-setting

I think adding the link/s to the first occurrence would be enough, right? 🤔

Reply:
Yes, linking once to a single page that includes all of the config options makes sense.